arkouda/pdarrayclass.py

```diff
@@ -1754,18 +1737,24 @@ def reshape(self, *shape, order="row_major"):
 
         Returns
         -------
-        ArrayView
-            An arrayview object with the data from the array but with the new shape
+        pdarray
+            a pdarray with the same data, reshaped to the new shape
         """
-        from arkouda.array_view import ArrayView
-
         # allows the elements of the shape parameter to be passed in as separate arguments
         # For example, a.reshape(10, 11) is equivalent to a.reshape((10, 11))
         if len(shape) == 1:
             shape = shape[0]
         elif not isinstance(shape, pdarray):
             shape = [i for i in shape]
-        return ArrayView(base=self, shape=shape, order=order)
```

Comment:
can the `order` argument be removed from reshape?

Reply:
Good catch!  

I think we should remove it temporarily, but add it back in later.  It does seem like it will be used in some of the libraries we want to integrated with, like `scipy`.  

I made a separate ticket.  https://github.com/Bears-R-Us/arkouda/issues/3729